README.md

```diff
@@ -142,6 +144,41 @@ convivaAnalytics.updateContentMetadata(metadata);
 
 Those values will be cleaned up after the session is closed.
 
+### Server Side Ad Tracking
+
+In order to track server side ads you can use the methods provided in `SsaiApi` which can be accessed via `ConvivaAnalytics.ssai`.
+The following example shows basic server side ad tracking:
+
+```swift
+convivaAnalytics.ssai.reportAdBreakStarted()
+
+const adInfo: SsaiAdInfo = {
+    title: "My ad title",
+    position: SsaiAdPosition.PREROLL,
+    duration: 30,
+}
+convivaAnalytics.ssai.reportAdStarted(adInfo)
+
+...
+
+convivaAnalytics.ssai.reportAdFinished()
+convivaAnalytics.ssai.reportAdBreakFinished()
+```
+
+In addition to the metadata provided in the `SsaiAdInfo` object at ad start, the following metadata will be auto collected from the main content metadata:
+
+- Stream URL
+- Asset Name
+- Is live flag
+- Defautl resource
```

Comment:
```suggestion
- Default resource
```

Reply:
oops, thanks fixed it by accepting the suggestion!